app/views/proposals/index.html.haml

```diff
@@ -67,20 +67,27 @@
                 %small.text-muted
                   = event.event_type.title
                   = "(#{event.event_type.length} min)"
-                  = "in #{event.track.name}" if event.track
+                  = "in #{event.traistck.name}" if event.track
```

Comment:
traistck ?

Reply:
Hey. 
Oops, It's a typos!

Reply:
@differentreality Should i make a pr for correcting this ?